tests/examples/source/dockerfiles/DockerfileWithWhitespace

```diff
@@ -0,0 +1,28 @@
+
+     
+
+   
+   FROM node:12
```

Comment:
Can you please elaborate this `DockerfileWithWhitespace` more? I mean the usecase.

Reply:
We were checking that this Dockerfile is still valid. We could remove it if this test is not needed?